algebra-utils/src/fft/domain/mod.rs

```diff
@@ -263,8 +263,11 @@ pub fn sample_element_outside_domain<
 #[cfg(test)]
 mod tests {
     use crate::get_best_evaluation_domain;
-    use crate::Field;
-    use crate::fields::mnt6753::fr::Fr;
+    use algebra::Field;
+    #[cfg(not(feature = "gpu"))]
```

Comment:
It's a little bit ugly... Can't we use the same test field for both CPU and GPU ?

Reply:
Ok, I've put BLS for both